howto-gallery/package.json

```diff
@@ -5,25 +5,53 @@
   "description": "Minecraft TypeScript How-to Gallery",
   "private": true,
   "devDependencies": {
-    "del": "^6.0.0",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.3.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-typescript": "^6.0.0-alpha.1",
+    "@minecraft/core-build-tasks": "../resources/minecraft-core-build-tasks-1.0.2-patch-34.tgz",
+    "eslint-plugin-minecraft-linting": "^1.0.1",
+    "npm-watch": "^0.11.0",
     "source-map": "^0.7.4",
-    "typescript": "^4.4.3"
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.2"
   },
   "scripts": {
+    "lint": "just-scripts lint",
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "package": "just-scripts package",
+    "local-deploy": "just-scripts local-deploy",
+    "mcaddon": "just-scripts mcaddon",
+    "watch": "npm-watch",
     "enablemcloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-1958404141-86561845-1752920682-3514627264-368642714-62675701-733520436",
     "enablemcpreviewloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-424268864-5579737-879501358-346833251-474568803-887069379-4040235476"
   },
+  "watch": {
+    "local-deploy": {
+      "patterns": [
+        "scripts",
+        "behavior_packs",
+        "resource_packs"
+      ],
+      "quiet": false,
+      "extensions": "ts,json,lang,png"
+    }
+  },
   "dependencies": {
-    "@minecraft/server": "^1.7.0-beta.1.20.40-preview.23",
-    "@minecraft/server-admin": "^1.0.0-beta.1.20.40-preview.21",
-    "@minecraft/server-gametest": "^1.0.0-beta.1.20.40-preview.21",
-    "@minecraft/server-net": "^1.0.0-beta.1.20.40-preview.21",
-    "@minecraft/server-ui": "^1.2.0-beta.1.20.40-preview.23",
-    "@minecraft/vanilla-data": "^1.19.80",
-    "decode-uri-component": "^0.2.2"
+    "@minecraft/math": "^1.1.0",
+    "@minecraft/server": "1.10.0-beta.1.20.70-preview.24",
+    "@minecraft/server-admin": "1.0.0-beta.1.20.70-preview.24",
+    "@minecraft/server-gametest": "1.0.0-beta.1.20.70-preview.25",
+    "@minecraft/server-net": "1.0.0-beta.1.20.40-preview.21",
+    "@minecraft/server-ui": "1.2.0-beta.1.20.70-preview.24",
+    "@minecraft/vanilla-data": "^1.20.60"
+  },
+  "overrides": {
+    "@minecraft/math": {
+      "@minecraft/server": "$@minecraft/server"
+    },
+    "@minecraft/server-gametest": {
+      "@minecraft/server": "$@minecraft/server"
+    },
+    "@minecraft/server-ui": {
+      "@minecraft/server": "$@minecraft/server"
+    }
```

Comment:
nice! I didn't know that syntax for referencing the dependency version. Is it NPM specific?

Reply:
Yes, it is a NPM feature: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides